utils/vara-ui/src/components/alert/alert.tsx

```diff
@@ -1,13 +1,14 @@
 import { CSSProperties, ReactNode } from 'react';
 import cx from 'clsx';
-import { ReactComponent as CrossSVG } from '../../assets/images/cross.svg';
+import CrossSVG from '../../assets/images/cross.svg?react';
 import { Button } from '../button';
 import styles from './alert.module.scss';
 
 type Options = {
-  type: 'info' | 'error' | 'loading' | 'success';
+  type: 'info' | 'error' | 'loading' | 'success' | 'notification-warning' | 'notification-info';
   style?: CSSProperties;
   title?: string;
+  footer?: string;
```

Comment:
```suggestion
  footer?: ReactNode;
```

Let's make it more flexible.
Also, probably better to specify `footer` in the `AlertType`, together with `content`.